src/aiperf/plot/core/aggregate_data_loader.py

```
@@ -0,0 +1,137 @@
+            return None
+
+        parsed_metrics: dict[str, ConfidenceMetricData] = {}
+        for name, entry in raw_metrics.items():
+            if not isinstance(entry, dict):
+                self.warning(
+                    lambda name=name: f"Skipping non-dict metric entry: {name}"
+                )
+                continue
+            try:
+                parsed_metrics[name] = ConfidenceMetricData(**entry)
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)